impl/src/main/java/dao/QuestionDao.java

```diff
@@ -33,29 +34,38 @@
             "answer_accepted, " +
             "question.title, " +
             "question.bounty, " +
-            "question.votes, " +
             "question.created_at, " +
             "question,user_id, " +
             "question.slack_id, " +
             "\"user\".picture, \"user\".name as created_by, " +
-            "(SELECT COALESCE(SUM(question_vote.value), 0) FROM question_vote WHERE question_vote.question_id = question.id) AS votes " +
+            "(SELECT COALESCE(SUM(question_vote.value), 0) FROM question_vote WHERE question_vote.question_id = question.id) AS votes," +
+            "(SELECT accepted_at FROM answer WHERE question_id = question.id AND accepted_at IS NOT NULL) AS accepted_at " +
         "FROM " +
             "question " +
         "INNER JOIN " +
             "\"user\" on \"user\".id = question.user_id " +
         "ORDER BY " +
-            "question.created_at DESC " +
+            "accepted_at DESC NULLS LAST, " +
+            "votes DESC NULLS LAST, " +
+            "question.created_at DESC, " +
+            "question.title " +
         "LIMIT " +
             ":limit")
     Observable<Question> getLatestQuestions(Integer limit);
 
+    /**
+     * Add new question
+     * @param userId
+     * @param question
+     * @param createdAt For testing purposes. Only used if non-null.
```

Comment:
If this is only used for testing, couldn't there be a separate call to overwrite createdAt for tests and leave production code free of special cases?